cypress/integration/single-planet.js

```diff
@@ -0,0 +1,53 @@
+describe('Single Planet Testing', () => {
+  const baseURL = 'http://localhost:3000/'
+
+  beforeEach(() => {
+    cy.fixture('testPlanets.json')
+      .then(planets => {
+        cy.intercept('https://dry-plains-91502.herokuapp.com/planets', {
+          body: planets
+        })
+      });
+    cy.visit(`${baseURL}`);
+  });
+  it('Should show a Name and Image for planet being visited', () => {
+    cy.visit(`${baseURL}mercury`)
+      .get('.planet-info-title').should('contain', 'Mercury')
+      .get('.planet-info-img').should('have.attr', 'src')
+        .then(alttext => {
+          expect(alttext).to.equal('../planet-pics/Mercury-pic.jpg')
+        })
+    cy.visit(`${baseURL}saturn`)
+      .get('.planet-info-title').should('contain', 'Saturn')
+      .get('.planet-info-img').should('have.attr', 'src')
+        .then(alttext => {
+          expect(alttext).to.equal('../planet-pics/Saturn-pic.jpg')
+        })
+  })
+  it('Should show fun facts for each planet being visited', () => {
+    cy.visit(`${baseURL}venus`)
+      .get('.planet-info-list').should('contain', 'Distance from the sun:')
+      .get('.planet-info-list').should('contain', 'Diameter:')
+      .get('.planet-info-list').should('contain', 'Length of day:')
+      .get('.planet-info-list').should('contain', 'Number of moons:')
+    cy.visit(`${baseURL}uranus`)
+      .get('.planet-info-list').should('contain', 'Mass:')
+      .get('.planet-info-list').should('contain', 'Gravity:')
+      .get('.planet-info-list').should('contain', 'Length of year:')
+  })
```

Comment:
I think this is the spot where you suggested adding some specific numbers to make sure that the right data is coming through and in the right format. I think that's a good idea!